testdata/cgo/main_cpp.cpp

```diff
@@ -0,0 +1,24 @@
+#include "main.h"
+
+class FortyThree {
+  int private_Member;
+  public:
+     FortyThree() {
+	private_Member = 43;
+     }
+     int get() {
+        return private_Member;
+     }
+};
+
+
+extern "C" int fortythree() {
+	FortyThree ft;
+
+	return ft.get();
+}
+
+extern "C" int other() {
+   return OTHER_CONSTANT;
```

Comment:
It looks like you're not checking the value here in main.go.

Reply:
Right! Improving this.